src/Sulu/Bundle/RouteBundle/Exception/RouteIsNotUniqueException.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\RouteBundle\Exception;
+
+use Sulu\Bundle\RouteBundle\Model\RoutableInterface;
+use Sulu\Bundle\RouteBundle\Model\RouteInterface;
+
+/**
+ * Route is not unique exception.
+ */
+class RouteIsNotUniqueException extends \Exception
```

Comment:
I know, but I read about it the last time, and it would be a good idea to do so 😄